### ISSUE TITLE
Don't assign aria- attributes to ComboBox if user does not specify them

### ISSUE THREAD

**Issue by qqli1209** [OfficeDev/office-ui-fabric-react#6600]
**Describe the feature that you would like added:** don't give a default value to aria-labelledby, aria-describedby or aria-owns to Combobox if user does not specify them.

**What component or utility would this be added to:** Combobox

**Have you discussed this feature with our team, and if so, who:** Yes. Our team has a bug that ARIA attributes must conform to valid values, and it's opened by accessibility reviewer. We hope that if not specify, don't give it an invalid aria- attributes.

**Additional context/screenshots**
![image](https://user-images.githubusercontent.com/13404320/46612398-fc8a4400-cb42-11e8-92ee-3e1b70b6272b.png)

We don't specify any element's id as **ComboBox198-option** or **ComboBox198-list**. But the component gives these aria- attributes by default.

In the source code of [ComboBox  ](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx)I find:
```
aria-labelledby={label && id + '-label'}
aria-label={ariaLabel && !label ? ariaLabel : undefined}
aria-describedby={keytipAttributes['aria-describedby']}
aria-owns={isOpen ? id + '-list' : undefined}
```

**Comment by qqli1209** [comment]
Is there anyone looking at the issue? :)

**Comment by jamesedjac** [comment]
Has any action been taken on this bug? This seems like a simple fix

**Comment by JasonGore** [comment]
I've created [a simple CodePen](https://codepen.io/jasongore/pen/rQNaZO) for reference. In your image it concerns me that `aria-labelledby` has no value, but I do not see that in this CodePen (image below.)

These ARIA values aren't invalid, they are pointing to other subcomponents of ComboBox. They're provided as a self-contained [combobox role](https://www.w3.org/TR/wai-aria-1.1/#combobox) identifying how the various subcomponents relate to each other in terms of accessibility. I've also run accessibility checks and they aren't being reported as invalid ARIA attributes that way either.

For example, attributes such as [`aria-owns`](https://www.w3.org/TR/wai-aria-1.1/#aria-owns) should be present when the DOM hierarchy cannot be used to represent the reationship between input and list, as is the case with `ComboBox`.

Can you expand on detail about what is invalid about these ARIA attributes? Thanks.

![image](https://user-images.githubusercontent.com/26070760/47939784-c9b73f80-dea5-11e8-9d9e-446e216f68b6.png)

**Comment by qqli1209** [comment]
@JasonGore  Thanks for your feedback. I can still see the invalid `aria-labeledby` and `aria-describedby` attributes  in our project. But in the [demo page](https://developer.microsoft.com/en-us/fabric#/components/ComboBox) of OUFR and your [code pen](https://codepen.io/jasongore/pen/rQNaZO), I cannot see the bug. It seems the version of OUFR in our project is too old. When we bump up to the latest version, the bug will go. I close the bug as it has been fixed in the latest version of OUFR.
/cc @jamesedjac
